source/_psr/psr-0-meta.twig

```diff
@@ -0,0 +1,12 @@
+---
+psr_number: psr-0
+permalink: /psr/psr-0/meta
+title: 'PSR-0 Meta Document'
+markdown_source: fig-standards/accepted/PSR-0.md
+related:
+    - { name: 'PSR-0: Autoloading Standard', permalink: /psr/psr-0/ }
+    - { name: 'PSR-4: Autoloading Standard', permalink: /psr/psr-4/ }
+    - { name: 'PSR-0 Meta Document' }
```

Comment:
Nice that we're referring to the superseeding documents! Maybe we should make clearer that these are the deprecated versions?

```suggestion
    - { name: 'PSR-0: Autoloading Standard (deprecated)', permalink: /psr/psr-0/ }
    - { name: 'PSR-4: Autoloading Standard', permalink: /psr/psr-4/ }
    - { name: 'PSR-0 Meta Document' }
```